### PR TITLE
Add Fleet Id as an input

### PR DESCRIPTION
**Pull request by backward-compatible**

### CODE REVIEW

**Review by ZhengzheYang — APPROVED** [review]
LGTM. Should we add usage info for this?

**Comment by backward-compatible** [comment]
> LGTM. Should we add usage info for this?

Just updated the usage with gcp vs non-gcp behavior. Lmk if I should add more info there.

File: asmcli/commands/help.sh [hunk 1]
```diff
@@ -62,11 +62,14 @@ OPTIONS:
   -l|--cluster_location  <LOCATION>   The GCP location of the target cluster.
   -n|--cluster_name      <NAME>       The name of the target cluster.
   -p|--project_id        <ID>         The GCP project ID.
-  --kc|--kubeconfig      <KUBECONFIG> Path to the kubeconfig file to use for CLI requests.
+  --kc|--kubeconfig <KUBECONFIG_FILE> Path to the kubeconfig file to use for CLI requests.
                                       Required if not supplying --cluster_location,
                                       --cluster_name, --project_id in order to locate
                                       and connect to the intended cluster.
   --ctx|--context        <CONTEXT>    The name of the kubeconfig context to use.
+  --fleet_id             <FLEET ID>   The ASM Fleet host project ID. Required for non-GCP
```
Comment (zerobfd) [inline asmcli/commands/help.sh:70]: Let's avoid the phrasing "ASM Fleet". In this instance I think we can just leave out the word "ASM" entirely and say "The Fleet host project ID."

If you know what at Fleet is it makes sense, if you don't then adding ASM isn't going to help.

File: asmcli/asmcli.sh [hunk 2]
```diff
@@ -1971,16 +1987,16 @@ is_membership_crd_installed() {
 }
 
 populate_environ_info() {
```
Comment (zerobfd) [inline asmcli/asmcli.sh:1989]: Since you were super detailed and renamed all of the other occurrences, can we rename this function as well?

**Review by zerobfd — APPROVED** [review]
We should change the usage wording to remove "ASM", otherwise it looks great. Thanks for being so thorough.

File: asmcli/commands/help.sh (hunk 1 above)
Comment (backward-compatible) [reply asmcli/commands/help.sh]: done

**Pull request closed without merging**